### PR TITLE
add SDK validate docs

### PR DESCRIPTION
**Pull request by dorschw**
related: [CIAC-7522](https://jira-dc.paloaltonetworks.com/browse/CIAC-7522)

### CODE REVIEW

**Review by YuvHayun — APPROVED** [review]

File: docs/concepts/demisto-sdk-validate.md [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+| IN123  | Checks if integration display name is versioned correctly, e.g.: ends with v<number>.                                            | Integration display names should end with 'v<number>' for version clarity and consistency.                                                                                                                                                                                                                                                                                                                                                                                                   | Yes           |
```
Comment (MosheEichler) [inline docs/concepts/demisto-sdk-validate.md:78]:
```suggestion
| IN123  | Checks if integration display name is versioned correctly, e.g.: ends with v\<number>.                                            | Integration display names should end with 'v\<number>' for version clarity and consistency.                                                                                                                                                                                                                                                                                                                                                                                                   | Yes           |
```

File: docs/concepts/demisto-sdk-validate.md [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+|--------|-------------------------------------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|---------------|
+| SC100  | Checks if script name is versioned correctly, e.g.: ends with V<number>.      | This standardization ensures consistency across content items.                                                                                                                                                                                                                                                                                                                                                                          | Yes           |
```
Comment (MosheEichler) [inline docs/concepts/demisto-sdk-validate.md:148]:
```suggestion
| SC100  | Checks if script name is versioned correctly, e.g.: ends with V\<number>.      | This standardization ensures consistency across content items.                                                                                                                                                                                                                                                                                                                                                                          | Yes           |
```

**Review by MosheEichler — APPROVED** [review]
Nice

Please see my comments

**Comment by tschanfeld** [comment]
@dorschw Should this table be added to https://docs-cortex.paloaltonetworks.com/r/1/Demisto-SDK-Guide/validate

**Pull request merged**